### PR TITLE
assistenza form

### PR DESCRIPTION
**Pull request by emnlpstr**
## Description

As already mentioned in #ISSUE_NUMBER, this PR tackles:
* ...
* ...
* ...

In particular, the ...

## Checklist

- [x] I followed the indications in the [CONTRIBUTING](CONTRIBUTING.md)
- [x] The documentation related to the proposed change has been updated accordingly (also comments in code).
- [x] Have you written new tests for your core changes, as applicable?
- [x] Have you successfully ran tests with your changes locally?
- [x] Ready for review! :rocket:

## Fixes

- Fixes #

### CODE REVIEW

File: contents/seo.yml [hunk 1]
```diff
@@ -33,7 +33,10 @@ unsubscribePage:
   description: Annulla la tua email per ricevere agli aggiornamenti dalla piattaforma PA Digitale 2026
 projectsSolutionPage:
   title: Presentazione progetti - PA digitale 2026
-  description: Per le misure con una platea ristrettadi beneficiari (fino a  1.000 PA), è previstauna modalità di accesso attraversola presentazione di progetti, sempre ispirata ai principi della semplicità e della riduzione degli oneri amministrativi.  
+  description: Per le misure con una platea ristrettadi beneficiari (fino a  1.000 PA), è previstauna modalità di accesso attraversola presentazione di progetti, sempre ispirata ai principi della semplicità e della riduzione degli oneri amministrativi.
+assistenzaPage:
+  title: Assistenza - PA digitale 2026
+  description: -----
```
Comment (sebbalex) [inline contents/seo.yml:39]: please fix the meta description field

File: src/components/modal/ModalMessage.jsx [hunk 2]
```diff
@@ -571,14 +571,14 @@ export const ModalMessage = () => {
...
-            <img className={classes.spinner} src="/assets/spinner.gif"></img>
+            <img className={classes.spinner} src="/assets/spinner.gif" alt=""></img>
           </div>
         </ModalFooter>
       </Modal>
       <div className="container test-docs">
         <div className="row">
           <div className="col-12 col-md-6">
-            <div className={classes.notification} role="alert" aria-labelledby="not2dms-title" id="not2dms2">
+            {/* <div className={classes.notification} role="alert" aria-labelledby="not2dms-title" id="not2dms2">
```
Comment (sebbalex) [inline src/components/modal/ModalMessage.jsx:581]: please remove the commented snippet

File: src/components/modal/ModalMessage.jsx [hunk 3]
```diff
@@ -571,14 +571,14 @@ export const ModalMessage = () => {
             <Button color="primary" type="submit" form="message-form">
               {sendButtonLabel}
             </Button>
-            <img className={classes.spinner} src="/assets/spinner.gif"></img>
+            <img className={classes.spinner} src="/assets/spinner.gif" alt=""></img>
```
Comment (sebbalex) [inline src/components/modal/ModalMessage.jsx:574]: `alt` should be filled with text

File: src/layouts/AssistenzaPage.jsx [hunk 4]
```diff
@@ -0,0 +1,839 @@
...
+              id="assistance-form"
+              target="formFrame"
+              onSubmit={formHandler}
+            >
+              <input
+                type="hidden"
+                name="captcha_settings"
+                value='{"keyname":"DEV1","fallback":"true","orgId":"00D3N0000004K3l","ts":""}'
+              />
+              <input type="hidden" name="debugEmail" value="mattia.g.puggioni@accenture.com" />
```
Comment (sebbalex) [inline src/layouts/AssistenzaPage.jsx:615]: this should be removed as well

File: src/layouts/AssistenzaPage.jsx [hunk 5]
```diff
@@ -0,0 +1,839 @@
...
+                    />
+                    <select
+                      className="d-none"
+                      id="00N3N00000GCzFW"
+                      name="00N3N00000GCzFW"
+                      title="Misura"
+                      required={true}
+                      onInvalid={customInvalid}
+                    >
+                      <option value="1.2 Abilitazione e facilitazione migrazione al Cloud">
```
Comment (sebbalex) [inline src/layouts/AssistenzaPage.jsx:716]: some measures are missing here. Probably we should consider to load them from yaml file and then iterate here.

File: src/layouts/AssistenzaPage.jsx [hunk 6]
```diff
@@ -0,0 +1,839 @@
...
+                    transform="rotate(45 17.3242 0.5)"
+                    fill="#5C6F82"
+                  />
+                  <rect y="1.56055" width="1.49987" height="24.4978" transform="rotate(-45 0 1.56055)" fill="#5C6F82" />
+                </svg>
+              </button>
+            </div>
+          </Col>
+        </Row>
+        <script src="https://www.google.com/recaptcha/api.js"></script>
```
Comment (sebbalex) [inline src/layouts/AssistenzaPage.jsx:832]: we shouldn't need this as we have the library for that.

**Review by sebbalex — CHANGES_REQUESTED** [review]

File: package.json [hunk 7]
```diff
@@ -33,6 +33,7 @@
     "prop-types": "^15.7.2",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-google-recaptcha": "^2.1.0",
```
[the remark this answers is not in the document]
Comment (emnlpstr) [reply package.json:36]: Because the v3 library is not compatible with salesforce

File: src/layouts/AssistenzaPage.jsx (hunk 6 above)
Comment (emnlpstr) [reply src/layouts/AssistenzaPage.jsx]: I removed it

**Pull request merged**